src/components/Availability.js

```diff
@@ -0,0 +1,39 @@
+import { Icon } from "@iconify/react";
+import { StyledAvailability } from "./styles";
+
+export default function Availability({ brewery }) {
+  return (
+    <StyledAvailability>
+      <p
+        aria-label={
+          brewery === "planning" ? "is available" : "is not available"
+        }
+      >
+        planning:
+        {brewery === "planning" ? (
+          <Icon icon="fluent:presence-available-10-regular" />
```

Comment:
I see `fluent:presence-available-10-regular` a few times, you could make this a variable above describing what it is e.g. `iconPresenceAvailable` and so on, and then use it here and in the other occurances.